<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confirmWidget</class>
 <widget class="QWidget" name="confirmWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#confirmWidget{background-color:rgb(245, 245, 245);border:1px solid #979797}</string>
  </property>
  <widget class="QLabel" name="m_imageLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>33</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_warningLabel">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>32</y>
     <width>228</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 20px &quot;Arial&quot;;
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_infoLabel">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>63</y>
     <width>425</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 17px &quot;Arial&quot;;
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelButtton">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>110</y>
     <width>68</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(7, 7, 7);font: 14px &quot;Arial&quot;;color: rgb(250, 250, 250);}
QPushButton:pressed {background-color: rgb(245, 193, 65);}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_OKButton">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>110</y>
     <width>68</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 94, 144);
font: 14px &quot;Arial&quot;;
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_closeButton">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>110</y>
     <width>68</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 94, 144);
font: 14px &quot;Arial&quot;;
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
